wai-extra/Network/Wai/Middleware/Autohead.hs

```diff
@@ -7,9 +7,7 @@ import Network.Wai
 import Data.Monoid (mempty)
 
 autohead :: Middleware
-autohead app req sendResponse
-    | requestMethod req == "HEAD" = app req { requestMethod = "GET" } $ \res -> do
-        let (s, hs, _) = responseToStream res
-        sendResponse $ responseBuilder s hs mempty
-    | otherwise = app req sendResponse
+autohead app req sendResponse = app req sendResponse
```

Comment:
Thanks for pointing this out @snoyberg. Moving too fast this morning. I have updated the PR to reflect this. Thanks!